internal/tools/orchestrator/scheduler/instanceinfo/instance.go

```diff
@@ -77,7 +78,8 @@ func (r *InstanceReader) ByEdgeSite(name string) *InstanceReader {
 	return r
 }
 func (r *InstanceReader) ByApplicationID(id string) *InstanceReader {
-	r.conditions = append(r.conditions, fmt.Sprintf("application_id = \"%s\"", id))
+	r.conditions = append(r.conditions, "application_id = ?")
+	r.values = append(r.values, id)
 	return r
 }
```

Comment:
how about `ByNamespace`, `ByNamespace` and other conditions

Reply:
All `ByXXX` method should be fixed.

Reply:
All `ByXXX` method should be fixed.